### ISSUE TITLE
negative box coords

### ISSUE THREAD

**Issue by fcakyon** [obss/sahi#159]
The above results are obtained using slice size of 512 but when I reduce slice size 256 an error occurred due to negative bounding box coordinates.

```
result = get_sliced_prediction(
    image,
    detection_model,
    slice_height = 256,
    slice_width = 256,
    overlap_height_ratio = 0.2,
    overlap_width_ratio = 0.2
)
```
.
.
.
.

/usr/local/lib/python3.7/dist-packages/sahi/annotation.py in __init__(self, box, shift_amount)
     31         """
     32         if box[0] < 0 or box[1] < 0 or box[2] < 0 or box[3] < 0:
---> 33             raise Exception("Box coords [minx, miny, maxx, maxy] cannot be negative")
     34         self.minx = int(box[0])
     35         self.miny = int(box[1])

Exception: Box coords [minx, miny, maxx, maxy] cannot be negative

_Originally posted by @Fetulhak in https://github.com/obss/sahi/issues/139#issuecomment-871710099_

**Comment by fcakyon** [comment]
@Fetulhak I have found the reason, will be fixed with https://github.com/obss/sahi/pull/160

**Comment by fcakyon** [comment]
@Fetulhak it is fixed in v0.5.2: https://github.com/obss/sahi/releases/tag/0.5.2 

Install latest version with `pip install -U sahi`
